lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -317,6 +323,36 @@ class LineChartPainter extends AxisChartPainter {
     }
   }
 
+  void drawExtraLines(Canvas canvas, Size viewSize) {
+    // Draw vertical data point lines under spots
+    viewSize = getChartUsableDrawSize(viewSize);
+    for (LineChartBarData barData in data.lineBarsData) {
+      if (barData.belowBarData.verticalLines == null) {
```

Comment:
why you are reading it from belowBarData?
dude draw it just like the horizontal lines,
you should have a class like this:
```
class ExtraLinesData {
  final bool show;

   // Horizontal line definitions
  final List<HorizontalLine> horizontalLines;

  // Vertical line definitions
  final List<VerticalLine> verticalLines;

   const ExtraLinesData({
    this.show = true,
    this.horizontalLines,
    this.verticalLines,
  });
}
```

both of them should define in this class

Reply:
You told me to move it to below bar data...